lua/modules/linear_bolzmann_solvers/discrete_ordinates_solver/lbs_compute_leakage.cc

```diff
@@ -49,4 +48,66 @@ chiLBSComputeLeakage(lua_State* L)
   return 1;
 }
 
+RegisterLuaFunction(ComputeLeakage, lbs, ComputeLeakage);
+
+int
+ComputeLeakage(lua_State* L)
+{
+  const auto fname = "lbs.ComputeLeakage";
+  const auto num_args = lua_gettop(L);
+
+  // Get the solver
+  LuaCheckNilValue(fname, L, 1);
+  const auto solver_handle = lua_tointeger(L, 1);
+  const auto& solver = opensn::Chi::GetStackItem<opensn::lbs::DiscreteOrdinatesSolver>(
+    opensn::Chi::object_stack, solver_handle, fname);
+
+  // Get the boundaries to parse
+  std::vector<uint64_t> bndry_ids;
+  if (num_args > 1)
+  {
+    LuaCheckTableValue(fname, L, 2);
+
+    // Define mapping from boundary name to id
+    const std::map<std::string, uint64_t> bndry_map = {
+      {"xmax", 0}, {"xmin", 1}, {"ymax", 2}, {"ymin", 3}, {"zmax", 4}, {"zmin", 5}};
+
+    // Get the boundaries
+    const auto n_bndrys = lua_rawlen(L, 2);
+    for (int b = 0; b < n_bndrys; ++b)
+    {
+      lua_pushinteger(L, b + 1);
+      lua_gettable(L, 2);
+      bndry_ids.push_back(bndry_map.at(lua_tostring(L, -1)));
+      lua_pop(L, 1);
+    }
+  }
+  else
+    bndry_ids = solver.Grid().GetDomainUniqueBoundaryIDs();
+
+  // Compute the leakage
+  const auto leakage = solver.ComputeLeakage(bndry_ids);
+
+  // Define mapping from boundary ids to names
+  const std::map<uint64_t, std::string> bndry_map = {
+    {0, "xmax"}, {1, "xmin"}, {2, "ymax"}, {3, "ymin"}, {4, "zmax"}, {5, "zmin"}};
```

Comment:
Similarly to the above comment, this might need to go into the `LBSSolver` as a public static, so it is consistent.